core/src/main/res/layout/dialog_add_note.xml

```diff
@@ -20,24 +20,30 @@
       android:layout_height="match_parent"
       android:orientation="vertical">
 
+      <HorizontalScrollView
```

Comment:
There are TextView attributes I believe we can use instead of a ScrollView.

Play around with `ellipsize=marquee` `scrollHorizontally` maybe

Reply:
@macgills I already tried that. That did not make the text view scrollable. 
Using this and making a few more changes, we can make it automatically scrollable, giving a marquee like effect. 
But I think for the notes dialog, a horizontal scroll view is better than a text view with a marquee effect.

Reply:
I am not convinced a user will scroll the title, I wouldn't anyway. I don't scroll the titles on a music app, I stare like an idiot at the marqueeing text trying to figure out what the hell the song is. If you can find me a horizontally scrolling textview in a popular app I'll allow it but I don't want to recreate a marquee with a scrollview.

Otherwise I'd prefer if we shrunk the title size, let it go to 3 lines max and if they don't have the full title after that then I don't care.

Reply:
Shrinking the text size and letting it go to 3 lines is a better solution here, I think. 
I'll update the pr accordingly.

Reply:
@macgills done.